x/incentives/keeper/distribute_test.go

```diff
@@ -161,6 +166,8 @@ func (suite *KeeperTestSuite) TestGetModuleDistributedCoins() {
 	suite.Require().Equal(coins, distrCoins)
 }
 
+// TODO: Is this testing the creation of a perp gauge that has no locks associated, so when its distribution
+// occurs it should still be an active gauge?
```

Comment:
TODO: Is this testing the creation of a perp gauge that has no locks associated, so when its distribution occurs it should still be an active gauge?

Reply:
See my reply below – I believe it's testing to make sure that if there are no locks associated, no tokens are distributed